decompose/src/commonMain/kotlin/com/arkivanov/decompose/Annotations.kt

```diff
@@ -22,3 +22,11 @@ annotation class ExperimentalDecomposeApi
 @RequiresOptIn(level = RequiresOptIn.Level.WARNING)
 @Retention(AnnotationRetention.BINARY)
 annotation class FaultyDecomposeApi
+
+/**
+ * Marks delicate Decompose API that requires special attention when used.
+ * See the docs of the annotated API for more information.
+ */
+@RequiresOptIn(level = RequiresOptIn.Level.WARNING)
```

Comment:
I'd recommend marking the level as ERROR to force people to read the description. Otherwise the issue may persist

Reply:
Thanks. I will keep it as warning for now, similar to DelicateCoroutinesApi.